editor/test/integration/extension_texturepacker_test.clj

```diff
@@ -377,7 +378,7 @@
     (let [node-id (test-util/resource-node project "/examples/basic/basic.tpatlas")]
       (with-open [_ (test-util/build! node-id)]
         (let [built-textureset (->> node-id test-util/node-build-output (protobuf/bytes->map-without-defaults TextureSetProto$TextureSet))
-              built-texture (->> built-textureset :texture (workspace/build-path workspace) fs/read-bytes (protobuf/bytes->map-with-defaults Graphics$TextureImage))]
+              built-texture (->> built-textureset :texture (workspace/build-path workspace) fs/read-bytes TextureUtil/byteArrayToTextureImage (protobuf/bytes->map-with-defaults Graphics$TextureImage))]
```

Comment:
We can use `protobuf/pb->map-with-defaults` here if we change `TextureUtil/byteArrayToTextureImage` to return a `TextureImage` instead of a byte array.